src/explore-education-statistics-common/src/modules/release/components/BasicReleaseSummary.tsx

```diff
@@ -0,0 +1,127 @@
+import FormattedDate from '@common/components/FormattedDate';
+import {
+  formatPartialDate,
+  isValidPartialDate,
+} from '@common/utils/date/partialDate';
+import SummaryList from '@common/components/SummaryList';
+import SummaryListItem from '@common/components/SummaryListItem';
+import ButtonText from '@common/components/ButtonText';
+import InfoIcon from '@common/components/InfoIcon';
+import useToggle from '@common/hooks/useToggle';
+import { Release } from '@common/services/publicationService';
+import {
+  releaseTypeComponents,
+  releaseTypes,
+} from '@common/services/types/releaseType';
+import { Dictionary } from '@common/types';
+import Modal from '@common/components/Modal';
+import { parseISO } from 'date-fns';
+import React, { ReactNode } from 'react';
+
+interface ReleaseTypeIcon {
+  url: string;
+  altText: string;
+}
+
+const nationalStatisticsLogo: ReleaseTypeIcon = {
+  url: '/assets/images/UKSA-quality-mark.jpg',
+  altText: 'UK statistics authority quality mark',
+};
+
+const releaseTypesToIcons: Dictionary<ReleaseTypeIcon> = {
+  NationalStatistics: nationalStatisticsLogo,
+};
+
+interface Props {
+  isEditing?: boolean;
+  lastUpdated?: Date;
+  release: Release;
+  releaseDate: string;
+  renderReleaseNotes: ReactNode;
+  renderStatusTags: ReactNode;
+  renderSubscribeLink: ReactNode;
+  onToggleReleaseTypeModal?: () => void;
```

Comment:
`onShowReleaseTypeModal` might be better. `onToggle` implies the event should fire when the modal closes too.